python/Ganga/testlib/GangaUnitTest.py

```diff
@@ -48,6 +48,7 @@ def start_ganga(gangadir_for_test, extra_opts=[]):
         ('Configuration', 'gangadir', gangadir_for_test),
         ('Configuration', 'user', 'testframework'),
         ('Configuration', 'repositorytype', 'LocalXML'),
+        ('Configuration', 'UsageMonitoringMSG', False),  # Turn off spyware
```

Comment:
Yes, but we're not currently draining this queue. I don't think this is a problem but if anyone complains we should turn it back off because it would require setting up another collector instance for draining the tests which is a lot of effort for little to no gain